src/metatensor/models/experimental/soap_bpnn/trainer.py

```diff
@@ -52,7 +52,7 @@ def train(
         assert len(devices) == 1
         device = devices[0]
 
-        logger.info(f"training on device {device} with dtype {dtype}")
+        logger.info(f"Training on device {device} with dtype {dtype}")
```

Comment:
Do we want to move this into the cli train function? in the train function we also set these values.

Reply:
do as you wish, what do @frostedoyster thinks?

Reply:
I would keep this in the architectures for now. And yes, let's capitalize the t

Reply:
btw this is in soap bpnn

Reply:
Thanks, I'll open a different PR

Reply:
I still changed it no worries